std/random.d

```diff
@@ -1336,6 +1330,297 @@ version (AnyARC4Random)
     extern(C) private @nogc nothrow
     {
         uint arc4random() @safe;
+        void arc4random_buf(scope void* ptr, size_t len) @system;
+    }
+
+    /+
+    Tries to use a system API to get entropy. Resources are not cached
+    due to the expectation that this function will not be called more
+    than once per program execution.
+
+    Params:
+        ptr = address at which to start writing
+        len = number of bytes to write
+
+    Returns:
+    always `len` (does not fail)
+    +/
+    private ptrdiff_t oneTimeGenRandom()(scope void* ptr, ptrdiff_t len) @system
+    {
+        arc4random_buf(ptr, len);
+        return len;
+    }
+}
+else version (Windows)
+{
+    import core.sys.windows.windows:
+        BOOL, HMODULE, FARPROC, LPCSTR, LPCWSTR, PBYTE,
+        ULONG_PTR, FreeLibrary, GetProcAddress, LoadLibraryA;
+    import core.sys.windows.wincrypt : HCRYPTPROV;
+    private alias DWORD = size_t; // uint in druntime
```

Comment:
Maybe we should finally fix druntime :laughing: 

The reason I added this was that the build was segfaulting otherwise on AppVeyor, but my Window-foo is limited to seeing things crash on Windows.

Reply:
For some reason I thought it had already been fixed, but it seems not.